storage/eoshomewrapper/eoshomewrapper.go

```diff
@@ -96,6 +103,17 @@ func (w *wrapper) GetMD(ctx context.Context, ref *provider.Reference, mdKeys []s
 		return nil, err
 	}
 
+	// We check that the resource is stored under an allowed path
```

Comment:
Shouldn't this be in reva upstream? The blocked path is CERN-specific, but being in a visible namespace, maybe not so much.